accordion/index.html

```diff
@@ -0,0 +1,126 @@
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+    <section>
+        <div class="accordion">Section 2</div>
+        <div class="segment">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+    <section>
+        <div class="accordion">Section 3</div>
+        <div class="segment">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam nec dictum ipsum, sed maximus massa.
+                Curabitur eleifend
+                lorem non posuere tristique. Maecenas vel porta augue. Phasellus lobortis ante id gravida egestas.
+                Pellentesque
+                facilisis ut odio in rhoncus. Duis ut sollicitudin massa. Aenean non nisi est. Nullam fermentum, sapien
+                posuere luctus
+                imperdiet, enim nisl vehicula dui, sit amet bibendum lectus risus in odio. Praesent sed sem tempor, pretium
+                orci in,
+                condimentum neque. Cras a magna eget erat tempus tincidunt ut quis enim. Praesent efficitur sollicitudin
+                lectus ac
+                vehicula
+            </p>
+        </div>
+    </section>
+
+    <script>
+        const accordion = document.getElementsByClassName("accordion");
+        const segment = document.getElementsByClassName("segment");
+        for (let i = 0; i < accordion.length; i++) {
+            accordion[i].addEventListener("click", function () {
+                this.classList.toggle("active");
+                const segment = this.nextElementSibling;
+                if (segment.style.display === "block") {
```

Comment:
Suggestion: We can use a class here and toggle that instead of applying inline CSS properties

Reply:
Woah, isn't this too late to merge now 😅